src/components/AreaSelect/AreaSelect.scss

```diff
@@ -0,0 +1,17 @@
+@import 'variables.scss';
```

Comment:
We don't want to be using sass anymore -- it's leftover from the Bulma days. Can you create a CSS file in `src/styles/components` and get rid of this .scss file?  Note that you'll also need to import your new css file in `src/styles/index.css`